gpjax/likelihoods.py

```diff
@@ -146,14 +143,14 @@ def predict(self, dist: tfd.Distribution) -> tfd.Distribution:
         return self.link_function(mean / jnp.sqrt(1.0 + variance))
 
 
-def inv_probit(x: Float[Array, "N 1"]) -> Float[Array, "N 1"]:
+def inv_probit(x: Float[Array, "*N"]) -> Float[Array, "*N"]:
```

Comment:
What does the `*` in `*N` do?

Reply:
like in `*args`, it gobbles up arbitrary dimensions. but beartype itself doesn't check the two are consistent